llama_index/llms/replicate.py

```diff
@@ -42,14 +44,16 @@ def __init__(
     def metadata(self) -> LLMMetadata:
         """LLM metadata."""
         return LLMMetadata(
-            context_window=self._context_window, num_output=self._max_tokens
+            context_window=self._context_window,
+            num_output=self._max_tokens,
+            model_name=self._model,
```

Comment:
I don't think this is a valid field for LLMMetadata?

Reply:
It is, merged a thingy today